azure-pipelines.yml

```
@@ -380,6 +374,89 @@ jobs:
       docker run $IMAGE_NAME /bin/bash -c "time az self-test && time az --version && sleep 5"
     displayName: 'Bash Script'
 
+- job: BuildExtraDockerImage
```

"Extra" is too general. It's a good practice to be specific.

```suggestion
- job: BuildAzureLinuxDockerImage
```